scripts/irods/test/test_iticket.py

```diff
@@ -1122,6 +1122,21 @@ def test_ticket_mtime_is_updated_when_ticket_properties_are_modified_by_non_admi
         self.assertGreater(modify_ts, create_ts)
         self.assertGreater(modify_ts, previous_ts)
 
+    def test_exceeding_ticket_uses_returns_correct_error_code__issue_7967(self):
+        ticket_string = "issue_7967_uses_count"
+        file_name = "issue_7967_uses_filename"
+        self.user.assert_icommand(['itouch', file_name])
+
+        self.user.assert_icommand(['iticket', 'create', 'read', file_name, ticket_string])
+
+        # Modify ticket usage amount
+        self.user.assert_icommand(['iticket', 'mod', ticket_string, 'uses', '3'])
+        self.user.assert_icommand(['iget', '-f', '-t', ticket_string, file_name])
+        self.user.assert_icommand(['iget', '-f', '-t', ticket_string, file_name])
+        self.user.assert_icommand(['iget', '-f', '-t', ticket_string, file_name])
+        self.user.assert_icommand(['iget', '-f', '-t', ticket_string, file_name],'STDERR', ['CAT_TICKET_USES_EXCEEDED'])
```

Comment:
Add other parts of the expected error message to future proof the test from things like `BLAH_CAT_TICKET_USES_EXCEEDED_BLAH`. Including the error code with its symbolic name is a good way to achieve that.